src/components/mapPageComponents/mapContainer/ducks/types.ts

```diff
@@ -0,0 +1,76 @@
+import { C4CState } from '../../../../store';
+import { ThunkAction } from 'redux-thunk';
+import { MapActions } from './actions';
+import { ApiExtraArgs } from '../../../../api/apiClient';
+import { AsyncRequest } from '../../../../utils/asyncRequest';
+
+// ---------------------------------Blocks----------------------------------------
+
+export interface BlockGeoData {
+  type: string;
+  name: string;
+  features: BlockFeatureResponse[];
+}
+
+interface BlockFeatureResponse {
+  type: string;
+  properties: BlockFeaturePropertiesResponse;
+  geometry: MapGeometry;
+}
+
+interface BlockFeaturePropertiesResponse {
+  block_id: number; // does not match linting
```

Comment:
❓

Reply:
"block_id" is returned from the backend with that casing and the linting rules we have don't like it. I can make comment more specific

Reply:
ah I see. In this case, we should use lowerCamelCase for the interface, and in the API client we just reassign the response values to lowerCamelCase.